src/elements/content-sidebar/activity-feed/annotation-thread/useAnnotationAPI.js

```diff
@@ -34,26 +34,30 @@ type UseAnnotationAPI = {
     handleDelete: ({ id: string, permissions: AnnotationPermission }) => any,
     handleEdit: (id: string, text: string, permissions: AnnotationPermission) => void,
     handleStatusChange: (id: string, status: FeedItemStatus, permissions: AnnotationPermission) => void,
+    initialReplies: Array<Comment>,
     isLoading: boolean,
 };
 
 const useAnnotationAPI = ({ annotationId, api, fileId, filePermissions, errorCallback }: Props): UseAnnotationAPI => {
     const [annotation, setAnnotation] = React.useState();
+    const [initialReplies, setInitialReplies] = React.useState([]);
     const [error, setError] = React.useState();
     const [isLoading, setIsLoading] = React.useState(true);
 
     const getAnnotationSuccess = (fetchedAnnotation: Annotation): void => {
         setAnnotation(fetchedAnnotation);
+        if (fetchedAnnotation.replies) {
+            setInitialReplies(fetchedAnnotation.replies);
```

Comment:
We're duplicating state here by storing the `replies` array twice, once nested and once not. Can we check the `fetchedAnnotation.replies` attribute later when needed, instead?